quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -52,14 +53,14 @@ pub struct Limits {
     pub(crate) bidirectional_remote_data_window: InitialMaxStreamDataBidiRemote,
     pub(crate) unidirectional_data_window: InitialMaxStreamDataUni,
     pub(crate) max_open_bidirectional_streams: InitialMaxStreamsBidi,
-    pub(crate) max_open_local_unidirectional_streams: InitialMaxStreamsUni,
+    pub(crate) max_open_local_unidirectional_streams: LocalUniDirectionalLimit,
     pub(crate) max_open_remote_unidirectional_streams: InitialMaxStreamsUni,
     pub(crate) max_ack_delay: MaxAckDelay,
     pub(crate) ack_delay_exponent: AckDelayExponent,
     pub(crate) max_active_connection_ids: ActiveConnectionIdLimit,
     pub(crate) ack_elicitation_interval: u8,
     pub(crate) ack_ranges_limit: u8,
-    pub(crate) max_send_buffer_size: u32,
+    pub(crate) max_send_buffer_size: LocalMaxSendBufferSize,
```

Comment:
Is this one necessary since there is no `RemoteMaxSendBufferSize`? Maybe just call it `MaxSendBufferSize`? Or save this for a later PR